pages/sendtoken.php

```diff
@@ -31,11 +31,26 @@
 $ldap = "";
 $userdn = "";
 $token = "";
+$usermail = "";
 
 if (!$mail_address_use_ldap) {
-    if (isset($_POST["mail"]) and $_POST["mail"]) { $mail = strval($_POST["mail"]); }
-     else { $result = "mailrequired"; }
-}
+    if (isset($_POST["mail"]) and $_POST["mail"]) {
+        $mail = $_POST["mail"];
+        
+    } elseif (isset($_REQUEST["usermail"]) and $_REQUEST["usermail"]) {
+        ###Check if email-address is submitted via URL
+        $usermail = $_REQUEST["usermail"];
```

Comment:
Same here, can you use `strval(` for consistency please.

Reply:
Done in #411